ui/package.json

```
@@ -8,7 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "cypress:open": "cypress open",
-    "test": "next lint && jest",
+    "test": "next lint && jest && pretty-quick --check",
```

Minor quibbles, but:

* add a way to do the reformatting without a commit
* run the tests in ascending order of slowness so if it's going to fail it will fail faster

```suggestion
    "prettier": "pretty-quick --write",
    "test": "next lint && pretty-quick --check && jest",
```